electron/src/js/getLogger.ts

```diff
@@ -0,0 +1,20 @@
+import {LogFactory, LoggerOptions} from '@wireapp/commons';
```

Comment:
License header is missing

Reply:
Good catch! It's in now: https://github.com/wireapp/wire-desktop/commit/220497e81c0b3bb9e187117d10d0498294845e2d